src/network/networkd-network.c

```diff
@@ -204,6 +204,28 @@ int network_load_one(Manager *manager, const char *filename) {
 
         dropin_dirname = strjoina(name, ".network.d");
 
+        network = new0(Network, 1);
+        if (!network)
+                return log_oom();
+
+        r = config_parse_many(filename, NETWORK_DIRS, dropin_dirname,
+                              "Match\0",
+                              config_item_perf_lookup, network_network_gperf_lookup,
+                              CONFIG_PARSE_WARN | CONFIG_PARSE_RELAXED, network);
+        if (r < 0)
```

Comment:
hmm, what's the goal here. this means we'll parse all configuration files twice, opening them twice for that. I don't grok why this would really be desirable?

Reading everything twice sounds less than ideal to me...

Is this just done to avoid slightly confusing log msgs?